lib/src/line.rs

```diff
@@ -204,28 +208,33 @@ pub struct Info {
     pub direction: Direction,
 
     /// The bias state of the line.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
```

Comment:
Personal preference to skip optional fields in most cases, rather than printing out  a lot of `null` values, especially in a case like this where most of these options are missing on a lot of boards.

Reply:
Definitely - the configs are frequently sparse so skip empty options.
And requiring exhaustive field population is bad for backward compatibility - the client always has to assume that missing fields are None as it might be talking to an older version that doesn't know of that field.